src/scriptdev2/scripts/eastern_kingdoms/molten_core/boss_garr.cpp

```diff
@@ -16,8 +16,8 @@
 
 /* ScriptData
 SDName: Boss_Garr
-SD%Complete: 80
-SDComment: Firesworn erruption needs to be revisited
+SD%Complete: 95
+SDComment: 'Summon Player' missing
```

Comment:
Most bosses in MC have this ability, but no clue when it got added. ID 20477 is certainly a vanilla spell.